Makefile

```diff
@@ -193,7 +194,7 @@ check-shell-scripts:
 ### Mics. Rules ###
 
 clean:
-	$(RM) $(BUILDFILE_SERVER) $(BUILDFILE_NIGHTLY_SERVER) $(BUILDFILE_AD_SERVER) $(BUILDFILE_NIGHTLY_AD_SERVER) $(BUILDFILE_CLIENT) $(BUILDFILE_TOOLBOX)
+	$(RM) $(BUILDFILE_PREFIX)*
```

Comment:
This is ok, but as a general rule I prefer explicit `$(RM)` in makefiles. Using wildcard may have some unexpected side-effects; see [Wildcard-Pitfall](https://www.gnu.org/software/make/manual/html_node/Wildcard-Pitfall.html).

For now, lets approve this and see in the future if it needs refinements.

Reply:
@synarete thanks for your review! It is correct that wildcards can bring pitfalls with them  and often, using explicit names instead can be more robust, but as it happened, in my work on PR #107 , the pitfall/problem was due to the fact that the clean target did *not* use a wildcard .

Reply:
I'm OK with this for now. If it causes problems we can revisit it in the future in a more explicit way.